languages/javascript/exercises/concept/array-transformations/.docs/hints.md

```diff
@@ -0,0 +1,37 @@
+## 1. Double every single card
+
+- There is a [built-in][map_method_docs] method great for simple transforms like this.
```

Comment:
Would it perhaps make sense to also have the "method" text be linked to?

```suggestion
- There is a [built-in method][map_method_docs] great for simple transforms like this.
```

If you feel this is useful, this comment also applies to the other hints.

Reply:
I was merely following the pattern established by other exercises. @SleeplessByte Should we change this everywhere?  I do see some merit.

Reply:
I actually don't love how we obscure the names of the methods￼ here. I feel like in any other context the name of the method would be part of the link text. ￼


Is that done on purpose because of the intention that these are hints?

Reply:
The fact that there is a built-in method is the first hint. The name of the method is the implicit second hint. Giving away the name of the method removes the hit of endorphin someone would get if they choose _not_ to follow the link.